lib/configure.js

```diff
@@ -288,8 +287,9 @@ function configure (gyp, argv, callback) {
       argv.unshift(gyp_script)
 
       // make sure python uses files that came with this particular node package
-      var pypath = new PathArray(process.env, 'PYTHONPATH')
+      var pypath = process.env.PYTHONPATH ? [process.env.PYTHONPATH] : []
       pypath.unshift(path.join(__dirname, '..', 'gyp', 'pylib'))
+      process.env.PYTHONPATH = pypath.join(process.platform === 'win32' ? ';' : ':')
```

Comment:
When you say wrap in an `if`, what did you have in mind?

I can certainly do:
```js
var pypath = 'PYTHONPATH' in process.env ? [process.env.PYTHONPATH] : []
```

Or would you prefer something like:

```js
var pypath = [path.join(__dirname, '..', 'gyp', 'pylib')]
if ('PYTHONPATH' in process.env) {
  pypath.push(process.env.PYTHONPATH)
}
process.env.PYTHONPATH = pypath.join(win ? ';' : ':')
```
(I kinda prefer that over an unshift anyway tbh)